core/templates/pages/preferences-page/preferences-page.directive.html

```diff
@@ -26,7 +26,11 @@
             <img ng-if="$ctrl.profilePictureDataUrl"
                  ng-src="<[$ctrl.profilePictureDataUrl]>"
                  class="img-thumbnail protractor-test-custom-photo">
-            <img ng-if="!$ctrl.profilePictureDataUrl" ng-src="<[$ctrl.getStaticImageUrl('/general/no_profile_picture.png')]>" class="img-thumbnail">
+            <picture>
+              <source type="image/webp" ng-srcset="<[$ctrl.getStaticImageUrl('/general/no_profile_picture.png')]>">
```

Comment:
For this can we check if the browser supports webp or not ? and then choose which image to support.
But ig all the targeted browsers already support webp.

Reply:
We don't have a list of supported browsers sadly. The problem with WebP is sadly not supported for Safari (https://caniuse.com/#feat=webp) which I think we should support.

Reply:
You can pick any approach you want to. I think either using the `<picture>` in all the places or try to find if the WebP is supported and then picking up the correct version of the image. If you go with the second approach, I think that you can use the `getStaticImageUrl` to select which type of file should be used and just call it without the extension.